app-benchmarks/unigine-superposition/unigine-superposition-1.1.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit wrapper desktop xdg-utils
```

Comment:
Usually we sort inherits alphabetically, except when an eclass needs to be last according to its documentation.

```suggestion
inherit wrapper desktop xdg
```
If you inherit `xdg` you can drop `pkg_postinst` and `pkg_postrm`

Reply:
Thanks, should this rule be added to `pkgcheck scan`?

Reply:
Done 2262d333e